app/views/organisers/index.html.slim

```diff
@@ -5,6 +5,13 @@
       p
       p
       p
+      = form_tag organisers_path, :method => :post do
```

Comment:
don't forget to display errors when user couldn't be saved as email was empty/ invalid/ already taken